### PR TITLE
Add order column to characters table

### PR DESCRIPTION
**Pull request by r-o-b-o-t-o**
## Description

Add the `order` column that has recently been added to the `characters` table.

### CODE REVIEW

File: docs/characters.md [hunk 1]
```diff
@@ -164,10 +165,11 @@ This table holds vital static information for each character. It is used to crea
...
 [74]: #grantablelevels
-[75]: #creation_date
-[76]: #deleteInfos_Account
-[77]: #deleteInfos_Name
-[78]: #deleteDate
+[75]: #order
+[76]: #creation_date
+[77]: #deleteInfos_Account
+[78]: #deleteInfos_Name
+[79]: #deleteDate
```
Comment (Kitzunu) [inline docs/characters.md:172]: rename them all small letters and no \_

Only need to do it here or the links will ve broken

Comment (Tony931023) [reply docs/characters.md:172]: It is already in the list of updates of the character table, PR #584, thanks to @r-o-b-o-t-o  I took the description of the order column.

**Comment by Tony931023** [comment]
It is already in the list of updates of the character table, PR #584, thanks to @r-o-b-o-t-o I took the description of the order column.

**Comment by r-o-b-o-t-o** [comment]
> It is already in the list of updates of the character table, PR #584, thanks to @r-o-b-o-t-o I took the description of the order column.

Ah right, closing this one then

**Pull request closed without merging**